exporters/trace/README.md

```diff
@@ -1 +1,47 @@
 # Cloud Trace Exporter for OpenTelemetry
+
+  OpenTelemetry Google Cloud Trace Exporter allows the user to send collected traces to Google Cloud. 
+  
+  <a href="https://cloud.google.com/trace">Google Cloud Trace</a> is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
+
+## Setup
+  Google Cloud Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+  Not currently ready as a package to be imported. To use presently, one would have to clone this GitHub repo.
+## Usage
+  If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.  
+    
+  The TraceExporter constructor takes in three parameters: String for projectId, TraceServiceClient to create a traceServiceClient instance, and Map<String, AttributeValue> for the fixed attributes of a span.  
```

Comment:
I'd consider wrapping some of the Java words with a single ` like
`projectId: string` it makes it show up monospaced but not quite the same as the ``` with syntax highlighting and new lines.

Reply:
Agreed, types and variables should be surrounded with backticks

Reply:
makes sense, didn't know how to do that before! Done.